packages/jest-expo/README.md

```diff
@@ -46,6 +46,43 @@ To test specific platforms you can use the following extensions:
 - web: `-test.web.*`
 - Node: `-test.node.*`, `-test.web.*`
 
+### ⚙️ Configuring your preset
+
+When building a custom preset you may want to use some of features provided by this preset. You can access these features through the `jest-expo/config` directory.
+
+#### `getWatchPlugins(jestConfig)`
+
+When given an existing Jest config this will return the `watchPlugins` used in `jest-expo`. This reads the `projects` field to return watchPlugins for single-project and multi-project configs.
```

Comment:
```suggestion
When given an existing Jest config this will return the `watchPlugins` used in `jest-expo`. This reads the `projects` field to determine which watchPlugins to return for single-project and multi-project configs.
```